chef-apply.gemspec

```diff
@@ -48,7 +48,7 @@ Gem::Specification.new do |spec|
                                      # localization gem...
   spec.add_dependency "toml-rb" # This isn't ideal because mixlib-config uses 'tomlrb'
                                 # but that library does not support a dumper
-  spec.add_dependency "train" # remote connection management over ssh, winrm
+  spec.add_dependency "train", ">= 1.4.24" # remote connection management over ssh, winrm
```

Comment:
Are there any notes we can add to the commit message about why this minimum version?